### DISCUSSION TITLE
"Dependabot updates are paused" because "you haven't used Dependabot in a while"

### DISCUSSION THREAD

**Discussion by alexpdp7** [community/community#51668] [Code Security]
### Select Topic Area

Bug

### Body

Hi,

Dependabot updates to my repo https://github.com/alexpdp7/ragent/ show this message. The message prompts me to merge Dependabot PRs to resume, but I have no open Dependabot PRs.

I think this happens become commonly, I merge many Dependabot PRs into a main PR, then run `cargo update`, then prepare my repo for a release. As the original PRs are not merged, perhaps Dependabot thinks I'm not paying attention? (I am!)

Cheers,

Álex

**Comment by jeffwidman** [comment]
👋 Glad you like :dependabot: 

I think the problem here is that you're cherry-picking the commits into your final PR... but not the Dependabot PR's... so :dependabot: thinks there's no user activity on its PRs.

You could probably change the base on one or more of the Dependabot PR's after they're generated so you can merge that Dependabot PR into your staging branch? Then it'll look like activity to Dependabot and prevent getting paused.

To kickstart it again, you can still manually trigger version updates to run to get another PR or maybe toggle security updates off/on to unpause... I'm not sure, but that should get it going enough to unstuck that repo.

**Reply by alexpdp7** [reply]
I was thinking... why not hook into whatever creates the "Looks like x is up-to-date now, so this is no longer needed.", and consider that "activity"?

**Reply by jeffwidman** [reply]
I wasn't part of the subteam that worked on this feature, so I don't know all the deatils. But I do recall they considered this scenario and decided to exclude it on purpose. I suspect the rationale was because if someone is updating, but not using Dependabot, then it seems like a generally clear signal that they don't value Dependabot itself. I realize that isn't true for some power users like yourself, but one of the main motivations for auto-pause was user complaining about :dependabot: being too noisy, so when it seems like a repo isn't using Dependabot to actually handle the bumps, then having it auto-pause felt like the intuitive choice for the common case.

**Reply by alexpdp7** [reply]
That's cool. Everything below here is "subjective", so make of it what you will.

I've seen people "complain" that merging many Dependabot PRs is a chore. Esp. depending on your repo configuration, you might need to merge, wait for another Dependabot to rebase, then merge that, etc. If your builds are slow, this can be quite painful.

As I have a repo which requires 1h to build (it does crosscompilation using Qemu), I adopted the practice of batching Dependabot PRs a bit; I create a branch and merge a couple of Dependabot PRs. That's much more efficient for me. Another nice thing is that I do a merge squash and then the nice commit messages for Dependabot are combined. I also frequently push a few more commits with stuff required to make a new release, or other misc. maintenance tasks.

At some point in the future, you may want to consider having this "batching" functionality built in Dependabot (although GitHub could also implement this). Tell Dependabot to combine a few of its PRs into one. This doesn't sound like anything that might be close in your roadmap, but I think it'd be interesting to steer people into doing this manually (this might end up saving some resources to Dependabot!).

In any case, I think you've got this thought out, so no worries- once I know what I need to do, I think I can work around this without much trouble, so feel free to "archive" this conversation.

**Reply by jeffwidman** [reply]
You may wish to subscribe to this issue for updates:
* https://github.com/dependabot/dependabot-core/issues/1190

(and see the last few entries on that thread 🎉 )

**Reply by bomgar** [reply]
I have the same problem. Is there any way to manually reactivate it?
I use dependabot to receive notification about new version. But I don't merge individual PRs. The grouped feature would work.

**Reply by tyranron** [reply]
@bomgar yeah, there are some the ways: https://github.blog/2023-01-12-a-smarter-quieter-dependabot/#how-do-i-wake-dependabot-up

For me, updating the `dependabot.yml`  file did the trick.

**Reply by jpmckinney** [reply]
I'm facing this issue. There seems to be no way to unpause Dependabot alerts. I have no open Dependabot PRs. I tried disabling and re-enabling Dependency graph, Dependabot alerts, and Dependabot security updates. I tried doing @dependabot in a previously merged PR and also an unmerged PR (where I had merged manually only twice).

Is there no way to unpause Dependabot?
